packages/client-config/src/client-config-writer/client_config_to_mobile_legacy_converter.ts

```diff
@@ -74,6 +74,17 @@ export class ClientConfigMobileConverter {
           },
         },
       };
+      let oauth;
+      if (clientConfig.oauth) {
+        oauth = {
+          WebDomain: clientConfig.oauth.domain,
+          Scopes: clientConfig.oauth.scope,
+          SignInRedirectURI: clientConfig.oauth.redirectSignIn,
+          SignOutRedirectURI: clientConfig.oauth.redirectSignOut,
+          AppClientId: clientConfig.oauth.clientId,
+        };
+        authConfig.plugins.awsCognitoAuthPlugin.Auth.Default.OAuth = oauth;
```

Comment:
nit: are we sure that we only want to change the pointer? Or should we create a new object to be safe?

Reply:
I didn't get you. `oauth` is a new object.

Reply:
Is it necessary to create `oauth`?

Reply:
nopes, not necessary. Will remove it.